nsxt/gateway_common.go

```diff
@@ -656,10 +656,7 @@ func parseGatewayInterfacePolicyPath(path string) (bool, string, string, string)
 		return false, "", "", ""
 	}
 
-	isT0 := true
-	if segs[2] != "tier-0s" {
-		isT0 = false
-	}
+	isT0 := !(segs[2] != "tier-0s")
```

Comment:
possibly dumb question: would it be ok to rewrite this to:

`isT0 :=  (segs[2] == "tier-0s")`

This would be just for readability

Reply:
Yes, it should be! The above was auto-generated with `golangci-lint run --fix` based on the definitions to correct QF1007.